changelog.rst

```diff
@@ -2,6 +2,21 @@
 **CHANGELOG**
 ================
 
+*v1.2.2*
+============
+
+NEW FEATURE: Auto publish package support added
```

Comment:
@sunil-lakshman , it is not advisable to publish internal code handling and publishing process details publicly. It would be more appropriate to include a general description such as "minor bug fixes and code improvements."